code-challenges.js

```diff
@@ -29,17 +29,34 @@ const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
 // Pseudo code:
+// step 1: I want to write a code that will join the two arrays together using a + or a concat.
+// step 2: I want to find the length using the .length feature to count the total elements in the array.
+// step 3: write code console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
+// step 4: Output is 9
+console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
```

Comment:
This does produce the desired output, but it does not do what the prompt asks. It asks you to combine the 2 arrays, then get the new length. This code is adding the lengths of both arrays together. Subtle difference, but make sure to keep attention to detail.